core/src/accounts_hash_verifier.rs

```diff
@@ -106,55 +106,63 @@ impl AccountsHashVerifier {
             hashes,
             exit,
             fault_injection_rate_slots,
+            accounts_hash,
         );
 
-        Self::submit_for_packaging(accounts_package, pending_snapshot_package, snapshot_config);
+        Self::submit_for_packaging(
+            accounts_package,
+            pending_snapshot_package,
+            snapshot_config,
+            accounts_hash,
+        );
     }
 
-    fn verify_accounts_package_hash(accounts_package: &AccountsPackage) {
+    /// returns accounts hash
+    fn verify_accounts_package_hash(accounts_package: &AccountsPackage) -> Hash {
```

Comment:
nit: Can this be renamed to `calculate_accounts_hash` or `calculate_and_verify_accounts_hash`?

Reply:
done